test/integration/deep/norelay/norelay_test.go

```diff
@@ -160,3 +143,31 @@ func getDeployments(t *testing.T) map[string]string {
 
 	return deploys
 }
+
+func getPodIp(t *testing.T, ns, selector string) string {
+	t.Helper()
+	ip, err := TestHelper.Kubectl(
+		"", "-n", ns, "get", "po", "-l", selector,
+		"-o", "jsonpath='{range .items[*]}{@.status.podIP}{end}'",
+	)
+	if err != nil {
+		testutil.AnnotatedFatalf(t, "failed to retrieve pod IP",
+			"failed to retrieve pod IP: %s\n%s", err, ip)
```

Comment:
Do we want to output `ip` if `Kubectl` returns an error?

```suggestion
			"failed to retrieve pod IP: %s", err)
```